tty/pc-tty/ttypc.c

```diff
@@ -91,12 +91,18 @@ static void ttypc_poolthr(void *arg)
 
 			case mtDevCtl:
 				idata = ioctl_unpack(&msg, &req, &id);
-				if (id < NVTS) {
-					err = ttypc_vt_ioctl(ttypc->vts + id, ioctl_getSenderPid(&msg), req, idata, &odata);
+				if (id == 0 && req == KIOEN) {
```

Comment:
`id == 0` condition should be verified inside `if` block to avoid the risk of passing a KIOEN request to `ttypc_vt_ioctl`.